scripts/plot_dispatch.py

```diff
@@ -0,0 +1,130 @@
+import os
+import sys
+
+import matplotlib.dates as mdates
+import matplotlib.pyplot as plt
+import oemoflex.tools.plots as plots
+from oemof_flexmex.model_config import plot_labels, colors_odict
+import pandas as pd
+from addict import Dict
+
+def sum_demands(data, bus_name, demand_name):
+    d_demand = pd.DataFrame()
+    for i in data.columns:
+        if demand_name in i[1]:
+            d_demand[i] = data[i]
+            data.drop(columns=[i], inplace=True)
+    total_demand = d_demand.sum(axis=1)
+    data[(bus_name, bus_name + '-demand', 'flow')] = total_demand
+    return data
+
+
+# normal dispatch plot
+# plot one winter and one summer month
```

Comment:
This should be a docstring, following the line of the function definition:

```
def function(...):
    r"""
    Description
    """
```

Reply:
Done in 3d63fd1